theano/tensor/basic.py

```diff
@@ -3167,6 +3170,12 @@ def var(input, axis=None, keepdims=False):
 
     # return the mean sqr
     v = mean((centered_input ** 2), axis, keepdims=keepdims)
+
+    # use 'corrected_two_pass' algorithm
```

Comment:
you should add a reference to the tech report http://www.cs.yale.edu/publications/techreports/tr222.pdf